### PR TITLE
Add check to catch delinquent eng/common changes

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Add check to catch delinquent eng/common changes

### CODE REVIEW

**Comment by weshaggard** [comment]
@chidozieononiwu I think the CI didn't kick off correctly for this PR because the template isn't yet in the repo so the template parsing failed.

**Comment by chidozieononiwu** [comment]
Could you help approve @weshaggard

File: eng/pipelines/templates/jobs/archetype-sdk-client.yml [hunk 1]
```diff
@@ -99,6 +99,7 @@ jobs:
       name: ${{ parameters.LinuxPool }}
       vmImage:
     steps:
+      - template: /eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml
       - task: UsePythonVersion@0
```
Comment (weshaggard) [inline eng/pipelines/templates/jobs/archetype-sdk-client.yml:103]: nit: would be nice to have an empty line.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**